src/lib/graphs/utils/find-columns.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Created on 8/25/16.
+ */
+import splitHeadsFromRest from './split-heads-from-rest';
```

Comment:
If this function is only used by this file, I think it makes sense to include it as part of this file. It can be split out later when it becomes heavier or has external uses.